test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Alias.Tests.ps1

```diff
@@ -120,7 +120,7 @@ Describe "Export-Alias DRT Unit Tests" -Tags "CI" {
 		}
 		else
 		{
-			chmod 777 $fulltestpath
+			chmod u+rw $fulltestpath
```

Comment:
i expect this is the same, and execute permission is probably not needed, but `rw` will definitely be needed for cleanup - would `rw-rw-rw-` be more palatable?

Reply:
```suggestion
			chmod a+rw $fulltestpath
```